### PR TITLE
TC_00.001.09 Creating new item

### PR DESCRIPTION
**Pull request by NadinLi**
Create FreestyleProject9Test

### CODE REVIEW

**Comment by NadinLi** [comment]
TC_00.001.09 | Creating new item>Creating Freestyle project

File: src/test/java/school/redrover/FreestyleProject9Test.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                        getDriver().findElement(By.cssSelector("#main-panel >h1")).getText(),
+                        "Project FreeStyleProjectName");
+
+
+
+
+
+
+
+
```
Comment (VictoriaLema) [inline src/test/java/school/redrover/FreestyleProject9Test.java:40]: please,delete empty lines from 33 to 50

**Comment by NadinLi** [comment]
https://trello.com/c/pYlFrV9T/145-tc0000109-creating-new-itemcreating-freestyle-project

File: src/test/java/school/redrover/FreestyleProject9Test.java (hunk 1 above)
Comment (NadinLi) [reply src/test/java/school/redrover/FreestyleProject9Test.java:40]: i will, thank you)

**Comment by NadinLi** [comment]
[TC_00.001.09 | Creating Freestyle project](https://trello.com/c/pYlFrV9T/145-tc0000109-creating-freestyle-project)

File: src/test/java/school/redrover/FreestyleProject9Test.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+public class FreestyleProject9Test extends BaseTest {
+
+    @Test
+    public void testCreat() {
+        /*
+        1.Create a freestyle project with valid name
+        2.See created project on a Dashboard
+        3.Can go to created project configuration page
+         */
+
```
Comment (verafes) [inline src/test/java/school/redrover/FreestyleProject9Test.java:19]: I'd like to suggest a formatting adjustment in the test code. Please ensure there's one empty line before the test code and no empty lines after 'assert' statements to maintain a streamlined format. Thank you

**Review by verafes — APPROVED** [review]

File: src/test/java/school/redrover/FreestyleProject9Test.java (hunk 2 above)
Comment (NadinLi) [reply src/test/java/school/redrover/FreestyleProject9Test.java:19]: внесла изменения

**Review by Forostinko — APPROVED** [review]

**Pull request merged**